### PR TITLE
Remove hca helpers and any unused tests that were importing them

### PR DESCRIPTION
**Pull request by CBonade**
## Description
With Nightwatch no longer running in GHA and Jenkins, we are starting the process of removing unused files.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ashagross — APPROVED** [review]
LG2M!

**Review by bkjohnson — COMMENTED** [review]
Do we already have Cypress tests to replace these `save-in-progress` tests?

**Comment by CBonade** [comment]
> Do we already have Cypress tests to replace these `save-in-progress` tests?

Sure do! I wrote those in the prior step of our deprecation process. They've been running in CI for a few weeks now and haven't had any errors associated with them, so just trying to move out the old ones now. 4 total SIP Cypress specs

**Review by bkjohnson — APPROVED** [review]

**Review by mdewey — APPROVED** [review]
LGTM

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -647,9 +647,9 @@ jobs:
           ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
           env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
 
-      # ----------------
-      # | Notify Slack |
-      # ----------------
+      # -----------------
+      # | Notify Slack  a|
```
Comment (ddzz) [inline .github/workflows/continuous-integration.yml:651]:
```suggestion
      # | Notify Slack  |
```

**Review by alexandec — APPROVED** [review]

**Pull request merged**